lib/logstash/codecs/retriggerable_task.rb

```diff
@@ -0,0 +1,90 @@
+require "concurrent"
+
+module LogStash module Codecs class RetriggerableTask
+  class DummyListener
+    def timeout
+      STDOUT.puts "..... task timed out ....."
+    end
+  end
```

Comment:
Its here so people can use the block form and not need to supply a listener in the args.

But I did not design the block form usage very well.